console/src/schematic/services/ontology.tsx

```diff
@@ -155,14 +126,14 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
   const del = useDelete();
   const copy = useCopy();
   const snapshot = useRangeSnapshot();
-  const download = useDownload();
+  const exprt = useExport(resources[0].name);
```

Comment:
Typo, just call it `export`

Reply:
can we call it something else that doesn't look like a typo then